lib/user-interface/react/src/shared/modal/confirmation-modal.tsx

```diff
@@ -27,7 +27,7 @@ export type ConfirmationModalProps = {
     resourceName: string;
     onConfirm: () =>  MutationActionCreatorResult<any>;
     postConfirm?: CallbackFunction;
-    description?: string;
+    description?: any;
```

Comment:
What would description be if not a `string`?

Reply:
I am passing HTML for the diff in config

Updated to:
`description?: string | ReactElement;`